bin/search.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+BUCKET=$1
+QUERY=$2
+
+# graphite connection settings
+PORT=2003
+SERVER=127.0.0.1
+TIMEFORMAT=%R
+
+exec 3>&1 4>&2
+
+MEASUREMNT=`{ time node ~/S3/bin/search_bucket.js -a accessKey1 -k verySecretKey1 -b $BUCKET -q $QUERY -h 127.0.0.1 -p 80 1>&3 2>&4; } 2>&1`  # Captures time only.
```

Comment:
We should.. also, depends on ~/S3, which is probably not what we want.. this was a tool just to test stuff but is used on perf_test.py. 

I don't know but we can probably implement a Python to query S3 directly (using HTTP) to kick off a search,  what do you think?

Reply:
We could. We will have to figure out how to alter the boto3 params to add the query "search" parameter before sending the list_objects api call (because you need some s3 client tool to create the authentication header unless we make the bucket public). I haven't dug in -- http://boto3.readthedocs.io/en/latest/reference/core/resources.html#module-boto3.resources.params.  Might be easy. If not, we can use the s3 tool or just make the bucket public read using boto3 and then use http to make the listing call with the added "search" param and no authentication header.